biggerquery/cli.py

```diff
@@ -190,36 +238,188 @@ def _parse_args(project_name: Optional[str], operations: [str]) -> Namespace:
                              'Examples: 2019-01-01, 2020-01-01 01:00:00')
+    _add_parsers_common_arguments(parser)
+
     if project_name is None:
-        parser.add_argument('--root',
+        parser.add_argument('--project_package',
                             required=True,
-                            help='The root package of your project. '
-                                 'Required because project_setup.PROJECT_NAME not found.')
-    return parser.parse_args()
+                            type=str,
+                            help='The main package of your project. '
+                                 'Should contain project_setup.py')
+
+
+def _add_parsers_common_arguments(parser):
+    parser.add_argument('-c', '--config',
+                        type=str,
+                        help='Config environment name that should be used. For example: dev, prod.'
+                             ' If not set, default Config name will be used.'
+                             ' This env name is applied to all biggerquery.Config objects that are defined by'
+                             ' individual workflows as well as to deployment_config.py.')
+
+
+def _add_deploy_parsers_common_arguments(parser):
+    parser.add_argument('-a', '--auth-method',
+                        type=str,
+                        default='local_account',
+                        help="One of two authentication metho: "
+                             "local_account -- you are using credentials of your local user authenticated in gcloud; "
+                             "service_account -- credentials for service account are obtained from Vault. "
+                             "Default: local_account",
+                        choices=['local_account', 'service_account'])
+    parser.add_argument('-ve', '--vault-endpoint',
+                        type=str,
+                        help="URL of a Vault endpoint to get OAuth token for service account."
+                             " Required if auth-method is service_account."
+                             " If not set, will be read from deployment_config.py"
+                        )
+    parser.add_argument('-vs', '--vault-secret',
+                        type=str,
+                        help="Vault secret token."
+                             " Required if auth-method is service_account."
+                        )
+    parser.add_argument('-dc', '--deployment-config-path',
+                        type=str,
+                        help="Path to the deployment_config.py file."
+                             " If not set, {current_dir}/deployment_config.py will be used.")
+
+    _add_parsers_common_arguments(parser)
+
+def _create_deploy_parser(subparsers):
+    parser = subparsers.add_parser('deploy',
+                                   description='BiggerQuery CLI deploy command -- performs complete deployment: deploys DAG files to Composer and Docker images to Container Registry.')
```

Comment:
Could we remove from description for every command "BiggerQuery CLI {command_name} --command"?  For deploy command example: "Performs complete deployment: deploys DAG files to Composer and Docker images to Container Registry.